notes/NewSiteArchitecture.md

```diff
@@ -0,0 +1,174 @@
+# New GraphQL.org Site Architecture:
+
+## Index
+
+*Goal:* This is the landing page and is our opportunity to quickly capture attention and explain what GraphQL is and why you should care.
+
+*Timeframe:* Launch Mon, Sept 12th
+
+This page is effectively a marketing page for GraphQL and should be the visual, scrollable version of the "Introducing GraphQL" conference talks and should be rich with visual metaphor and illustration and take advantage of whitespace to make individual salient points.
+
+Above the fold, this page should succinctly explain what GraphQL is and illustrate with a simple (editable) query/response example. Before scrolling you should understand the following:
+
+* GraphQL is an query language for APIs.
+* GraphQL is sent by client applications, such as an iOS app.
+* GraphQL is evaluated by a web service and often returned as JSON.
+* GraphQL typically replaces REST.
+
+Below the fold we should introduce concepts one at a time, each with visual metaphor and take-aways:
```

Comment:
When talking to developers currently using and also looking into GraphQL, the main benefit for them was that it presents an API layer organized in terms of objects, types, and fields, rather than endpoints. So it might be worth moving "GraphQL services provide a type system." to the front, and emphasize that it:

1. Provides a complete description of the data available, without needing to write a separate type layer like RAML, Swagger, etc.
2. Saves you from writing new endpoints, which means your API remains consistent even when you have multiple clients.

I think those two resonated the most with people we talked to.